std/root.ln

```diff
@@ -129,6 +129,13 @@ export fn negate(n: int64) = negi64(n)
 export fn negate(n: float32) = negf32(n)
 export fn negate(n: float64) = negf64(n)
 
+export fn abs(n: int8) = absi8(n)
+export fn abs(n: int16) = absi16(n)
+export fn abs(n: int32) = absi32(n)
+export fn abs(n: int64) = absi64(n)
+export fn abs(n: float32) = absf32(n)
+export fn abs(n: float64) = absf64(n)
```

Comment:
A long time ago I had already defined these opcodes in the compiler, but forgot to implement them everywhere else, in case you're curious why `compile/src/lntoamm/opcodes.ts` didn't need a change.